packages/graphql-migrations/src/util/defaultNameTransforms.ts

```diff
@@ -0,0 +1,15 @@
+import { NameTransformDirection } from '../abstract/generateAbstractDatabase'
+
+export function defaultColumnNameTransform(name: string, direction: NameTransformDirection) {
+  if (direction === 'to-db') {
+    return `${name.charAt(0).toLowerCase()}${name.slice(1)}`
+  }
+  return name
+}
+
+export function defaultTableNameTransform(name: string, direction: NameTransformDirection) {
+  if (direction === 'to-db') {
```

Comment:
Ok so there is very clear way to determine column names and fields and we provide defaults, so if graphback will setup this values beforehand and use it everythwere then we do not need do to anything - we will have our own default way to map data using this functions. Right?

Reply:
Yes - unless the developer decides to customise a field name with annotations.